src/pytest_plugins/solc/solc.py

```diff
@@ -63,6 +65,12 @@ def pytest_configure(config: pytest.Config):
         except ArgumentTypeError:
             version = None
         if version != solc_version:
+            # solc-select current does not support ARM linux
+            if platform.system().lower() == "linux" and platform.machine().lower() == "aarch64":
+                error_message = f"Version {version} does not match solc_version {solc_version}\
+ and since solc-select currently does not support ARM linux we can not recover from this problem."
```

Comment:
Your last commit inadvertently deleted the clarification from the previous commit? 
```suggestion
 and since solc-select currently does not support ARM linux you must manually do the following: \
Build solc from source, and manually move the binary to\
 .venv/.solc-select/artifacts/solc-x.y.z/solc-x.y.z, then run 'uv run solc-select use <x.y.z>'.
```

Also for strings that don't fit in the allowed line-length, please don't use `\`, rather do one of the following:
1. Use brackets that contain (and concatenate) multiple single-line strings. E.g.,
    https://github.com/ethereum/execution-spec-tests/blob/c91549cf31c503d5ea9b3037093449dd0bb4d146/src/pytest_plugins/filler/filler.py#L93-L98
2. `textwrap.dedent()` with a multi-line string:
    https://github.com/ethereum/execution-spec-tests/blob/c91549cf31c503d5ea9b3037093449dd0bb4d146/src/pytest_plugins/consume/hive_simulators/conftest.py#L203-L219

For this case, I'd go for 1.